android/java/proguard.flags

```diff
@@ -4,6 +4,8 @@
 
 -keep class org.chromium.chrome.browser.BraveSyncInformers { *; }
 
+-keep class org.chromium.components.permissions.BravePermissionDialogDelegate { *; }
```

Comment:
@samartnik what about this line? added it just now. Will `changeMethodOwner` and `changeSuperName` work without proguard exclusions?

Reply:
Yes, they will. We need exceptions here only when we invoke methods in runtime dynamically.

Reply:
oh, I think I got it.
Is all this bytecode replacement stuff done at compile/build time? And because of that the proguard flags are not needed until we want access class or invoke method via reflection at runtime.
is it correct?

Reply:
Yep

Reply:
got it. removed unnecessary modifications to proguard.flags.